### PR TITLE
Add .expand utility function

### PR DESCRIPTION
**Pull request by vhf**
This is a pretty common operation for which a utility can be useful, see doc diff, examples or tests:

https://github.com/zazuko/rdf-vocabularies/blob/03b7938ff5b5d0fb0fc4c096ed478d1fb09bc4be/index.test.js#L50-L68

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +104,40 @@ rdfVocabularies({ only: ['rdfs', 'owl', 'skos'] })
...
+
+* `rdfVocabularies.expand(prefixedTerm, Array<String>): Promise<String>` **asynchronous**
+
+    Expand with type check:
+    ```js
+    const rdfVocabularies = require('@zazuko/rdf-vocabularies')
+    const Class = rdfVocabularies.expand('rdfs:Class')
+    const Property = rdfVocabularies.expand('rdf:Property')
+
+    // Will return `http://schema.org/Person` iff the dataset contains either
```
Comment (tpluscode) [inline README.md]:
```suggestion
    // Will return `http://schema.org/Person` if the dataset contains either
```

File: README.md [hunk 2]
```diff
@@ -102,6 +104,40 @@ rdfVocabularies({ only: ['rdfs', 'owl', 'skos'] })
...
+    // <schema:Person> <rdf:type> <rdfs:Class>
+    // or
+    // <schema:Person> <rdf:type> <rdf:Property>
```
Comment (tpluscode) [inline README.md]: Only class and property? What about datatypes?

File: index.js [hunk 3]
```diff
@@ -60,6 +63,45 @@ function loadFile (prefix, { customSelection, factory }) {
...
+  if (!baseIRI) {
+    throw new Error(`Unavailable prefix '${prefix}:'`)
+  }
+
+  const iri = `${baseIRI}${term}`
+  if (!types.length) {
+    return iri
+  }
+
+  return Promise.resolve()
```
Comment (tpluscode) [inline index.js]: How about extracting this to an `async function` to get rid of the promises?

```suggestion
  return expandWithCheck(prefixed, types)
```

**Review by tpluscode — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vhf) [reply README.md]: I really meant *iff* :)

File: README.md (hunk 2 above)
Comment (vhf) [reply README.md]: I never used those. Any suggestion on how to make this API accommodate them?

File: index.js (hunk 3 above)
Comment (vhf) [reply index.js]: Good idea!

File: README.md (hunk 2 above)
Comment (tpluscode) [reply README.md]: I think it would be enough to also look for `?t <rdf:type> <rdfs:Datatype>`

Comment (vhf) [reply README.md]: Yep, you could do this by passing your term to the 2nd argument: `await rdfVocabularies.expand('schema:Person', [Class, Property])`

**Review by tpluscode — APPROVED** [review]

**Review by bergos — APPROVED** [review]

**Review by ktk — APPROVED** [review]
Looks very useful thanks, especially the one where one with checks

**Pull request merged**